Watchman.Integrations/Images/ImagesService.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+
+namespace Watchman.Integrations.Images
+{
+    public class ImagesService
+    {
+        public IEnumerable<KeyValuePair<string,Stream>> GetImagesFromResources(Func<string, bool> selector = null)
```

Comment:
zastanawiam sie nad zrobieniem klasy Image
która na razie miałaby nazwe i ten stream 

możliwe że w przyszłości dojdzie potrzeba dodania takiej klasy
lepiej byłoby nie musieć przepisywać obecnego kodu w Watchman.Discord 

co prawda poprawienie tego kodu zajęłoby kilka sekund bo to szczegół 
hmm
nie wiem

Reply:
Myślę, że warto zrobić to na wzór response czyli dodać klasę, rozszerzalne + mały nakład pracy + "jednorodność"

Reply:
no tak
i nie będzie sytuacji że nagle trzeba zmienić to na klase bo sie rozrosło 
więc przepisujemy sporo kodu

będzie fajna kompatybilność